test/TestCliRolePaths.py

```diff
@@ -8,10 +8,12 @@ class TestCliRolePaths(unittest.TestCase):
     def setUp(self):
         self.local_test_dir = os.path.dirname(os.path.realpath(__file__))
 
-    def run_ansible_lint(self, cwd, role_path, bin=None, env=None):
+    def run_ansible_lint(self, cwd, role_path=None, bin=None, env=None):
         if not bin:
             bin = sys.executable + " -m ansiblelint"
-        command = '{} {}'.format(bin, role_path)
+        command = '{} -v {}'.format(
+            bin or (sys.executable + " -m ansiblelint"),
```

Comment:
This part doesn't make sense with the if-clause above. Either remove that block or this fallback.